src/libs/HttpUtils.js

```diff
@@ -94,10 +95,11 @@ function xhr(command, data, type = CONST.NETWORK.METHOD.POST, shouldUseSecure =
 
         formData.append(key, val);
     });
+
     let apiRoot = shouldUseSecure ? CONFIG.EXPENSIFY.SECURE_EXPENSIFY_URL : CONFIG.EXPENSIFY.URL_API_ROOT;
 
-    if (shouldUseSecure && shouldUseSecureStaging) {
-        apiRoot = CONST.STAGING_SECURE_URL;
+    if (CONFIG.IS_IN_STAGING && shouldUseStagingServer) {
```

Comment:
I think this is because if you are in production we dont want to allow you to use staging api. But if you are in staging you can use either @parasharrajat Can you elaborate on why it should not work?

Reply:
Oh my bad, I see. I got confused. We want users to choose the secure staging server on the PROD app not on the staging app.

Reply:
Oh, confusing.... 
![image](https://user-images.githubusercontent.com/24370807/195126107-dbd22a17-1c36-44e7-b989-19c4b981ccf2.png)


You are saying that user should be able to use staging on PROD.

Reply:
? I am confused I wrote:

> I think this is because if you are in production we dont want to allow you to use staging api

If you in production, you can ONLY use production API.

If you in staging App, you can use either and you decide using the toggle

Reply:
> You are saying that user should be able to use staging on PROD.

Quite opposite, the user should be able to use production/staging API on Staging Build.